app/views/receipts/index.html.erb

```diff
@@ -83,12 +83,15 @@
   </div>
 </div>
 
+
 <div class="ibox float-e-margins">
   <div class="ibox-content">
 
-    <table class="table table-striped cats-datatable">
+    <table   id="example" class="table">
```

Comment:
Here we have to use the common cats-datatable class defined in application.js, for the purpose of easy maintenance.